File: plugins/approvej-intellij-plugin/src/unitTest/java/org/approvej/intellij/ApproveReceivedActionTest.java
package org.approvej.intellij;

import static org.assertj.core.api.Assertions.assertThat;

import com.intellij.openapi.actionSystem.ActionUpdateThread;
import org.junit.jupiter.api.Test;

class ApproveReceivedActionTest {

  @Test
  void getActionUpdateThread() {
    assertThat(new ApproveReceivedAction().getActionUpdateThread())
        .isEqualTo(ActionUpdateThread.BGT);
  }
}

File: plugins/approvej-intellij-plugin/src/unitTest/kotlin/org/approvej/intellij/ApproveReceivedActionTest.kt
package org.approvej.intellij

import com.intellij.openapi.actionSystem.ActionUpdateThread
import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class ApproveReceivedActionTest {

  @Test
  fun getActionUpdateThread() {
    assertThat(ApproveReceivedAction().actionUpdateThread).isEqualTo(ActionUpdateThread.BGT)
  }
}
